### DISCUSSION TITLE
Cannot see Redshift Spectrum Tables under the Tables Node in the Navigator

### DISCUSSION THREAD

**Discussion by ANTGOMEZ** [dbeaver/dbeaver#14037] [General]
I have the latest version of Community Edition DBeaver, I also have the very latest AWS JDBC driver, from here: https://docs.aws.amazon.com/redshift/latest/mgmt/jdbc20-download-driver.html

I  create a new Redshift connection, expand a DB down to the Spectrum schema and expand the Tables node, **_no tables show up_**. The tables do exist, I can view them in another client navigator. Also, others that use DBeaver Community here can see the tables with same driver and version.

Any advice appreciated.

Thanks!

**Comment by LonwoLonwo** [comment]
https://github.com/dbeaver/dbeaver/issues/14039
